database/migrations/2023_11_28_103705_easymerchant.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use App\Models\Gateway;
+use App\Models\GatewayType;
+use Illuminate\Support\Str;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        $check_gateway = Gateway::find(63);
+            
+        if(!$check_gateway){
+            $gateway = new Gateway();
+            $gateway->id = 63;
+            $gateway->name = 'Easymerchant';
+            $gateway->key = Str::lower(Str::random(32));
```

Comment:
this needs to be a static string, otherwise it will be different on every installation.

Reply:
okay sure. let me make it to static data